indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -157,7 +161,18 @@ function EditableListDisplay({
         options: [
           {value: 'not_submitted', text: Translate.string('Not submitted'), color: 'default'},
           {value: 'ready_for_review', text: Translate.string('Ready for review'), color: 'grey'},
-          {value: 'accepted', text: Translate.string('Accepted'), exclusive: true, color: 'green'},
+          {
+            value: 'accepted',
+            text: Translate.string('Accepted'),
```

Comment:
Might be too nitpicky but it would be nice to add context to some of these strings
```suggestion
            text: Translate.string('Accepted', 'Editable'),
```